src/main/content/_assets/css/blog.scss

```diff
@@ -18,15 +18,28 @@
 
 #blog_container {
     padding-top: 190px;
+
+    @media (max-width: 767px) {
```

Comment:
We should be using max-wdith 767.98px from now on. See https://getbootstrap.com/docs/4.0/layout/overview/.